### ISSUE TITLE
Tabbing through entities

### ISSUE THREAD

**Issue by laurakalbag** [statelyai/feedback#299]
### Details
Users should be able to use the keyboard to tab between state nodes and edges (document order) in the canvas
https://discord.com/channels/795785288994652170/912333666456313907/912443169524826152
### Category

Editor

---
Originally posted on: 2022-06-20T22:54:32.608Z
By: David Khourshid
Previous votes: 1
Previous status: under review
